toolchain/sem_ir/inst_fingerprinter.cpp

```diff
@@ -73,11 +73,14 @@ struct Worklist {
       return;
     }
     const auto& entity_name = sem_ir->entity_names().Get(entity_name_id);
-    if (entity_name.bind_index.has_value()) {
-      Add(entity_name.bind_index);
+    if (entity_name.bind_index().has_value()) {
+      Add(entity_name.bind_index());
```

Comment:
Is it deliberate that if a bind_index and NameId match, these will fingerprint the same?